usecases/case_usecase.go

```diff
@@ -114,16 +113,12 @@ func (usecase *CaseUseCase) ListCases(
 	ctx context.Context,
 	organizationId string,
 	pagination models.PaginationAndSorting,
-	filters dto.CaseFilters,
+	filters models.CaseFilters,
 ) (models.CaseListPage, error) {
 	if !filters.StartDate.IsZero() && !filters.EndDate.IsZero() &&
 		filters.StartDate.After(filters.EndDate) {
 		return models.CaseListPage{}, fmt.Errorf("start date must be before end date: %w", models.BadParameterError)
 	}
-	statuses, err := models.ValidateCaseStatuses(filters.Statuses)
```

Comment:
Same remark with that. It that related to the UUID change at all? Or is a change of convenience or a side effect?